ratatool-sampling/src/test/scala/com/spotify/ratatool/samplers/BigSamplerTest.scala

```diff
@@ -354,10 +354,11 @@ object BigSamplerTest extends Properties("BigSampler") {
 
   property("hashes of bytes, hex strings, and fixed should be the same") = forAll(
     specificAvroGen) { case avro =>
+    val hexEncode = BaseEncoding.base16.lowerCase
```

Comment:
Why base16?

Reply:
I changed this a bit now so that you can select how you want to encode the bytes. Either raw, hex, or base64. The reason for hex here was for the particular use case I had, that in some datasets the same field is stored as hex strings in others as bytes, but I want to be able to sample consistently on them.